kmultiversx/src/kmultiversx/kasmer.py

```diff
@@ -226,16 +263,14 @@ def arg_types_to_strategy(types: Iterable[str]) -> SearchStrategy[tuple[str, ...
 # Hypothesis test runner
 
 
-def test_with_hypothesis(
-    krun: KRun, sym_conf: KInner, init_subst: dict[str, KInner], endpoint: str, arg_types: Iterable[str], verbose: bool
-) -> None:
+def test_with_hypothesis(krun: KRun, sym_conf: Pattern, endpoint: str, arg_types: Iterable[str], verbose: bool) -> None:
```

Comment:
Then this can also become:

```suggestion
def test_with_hypothesis(kprint: KPrint, sym_conf: Pattern, endpoint: str, arg_types: Iterable[str], verbose: bool) -> None:
```